### ISSUE TITLE
glfwGetError linking issue on master branch

### ISSUE THREAD

**Issue by mihiic** [raysan5/raylib#3470]
- [x] I tested it on latest raylib version from master branch
 - [x] I checked there is no similar issue already reported
 - [x] I checked the documentation on the [wiki](https://github.com/raysan5/raylib/wiki)
 - [x] My code has no errors or misuse of raylib

### Issue description

glfwGetError function causes linker error when compiling for Web using emcc.
```
wasm-ld: error: ../../raylib/src/libraylib.a(rcore.o): undefined symbol: glfwGetError
```

I suspect this file and line is causing problem:
rcore_web.c : line 867
```
result = glfwGetError(NULL);
```

Issue is not present in `4.5.0` version of the library, but is present on `master` branch, which makes sense because result = `glfwGetError(NULL)` function is not called in version 4.5.0

### Environment
Dockerfile for complete environment :)
```
FROM emscripten/emsdk:3.1.30

RUN apt update
RUN apt install libasound2-dev libx11-dev libxrandr-dev libxi-dev libgl1-mesa-dev libglu1-mesa-dev libxcursor-dev libxinerama-dev -y
RUN apt install build-essential git unzip -y

WORKDIR /
# RUN git clone https://github.com/raysan5/raylib.git --depth 1
RUN wget https://github.com/raysan5/raylib/archive/refs/tags/4.5.0.zip
RUN ls
RUN unzip 4.5.0.zip
RUN mv raylib-4.5.0 raylib
WORKDIR /raylib
RUN ls

WORKDIR /raylib/src
RUN make PLATFORM=PLATFORM_WEB
RUN make install PLATFORM=PLATFORM_WEB

WORKDIR /
COPY raylib-build-web.sh .
RUN chmod +x raylib-build-web.sh

CMD [ "/raylib-build-web.sh" ]
```
raylib-build-web.sh
```
#!/usr/bin/bash
cd /game/src
make PLATFORM=PLATFORM_WEB
```
pwd is just unmodified raylib-game-template
```
sudo docker run -v $(pwd)/src:/game/src raylib:web
```

### Code Example

Unmodified raylib game template.

### Workaround
Don't use latest version of raylib :)
```
FROM emscripten/emsdk:3.1.30

RUN apt update
RUN apt install libasound2-dev libx11-dev libxrandr-dev libxi-dev libgl1-mesa-dev libglu1-mesa-dev libxcursor-dev libxinerama-dev -y
RUN apt install build-essential git unzip -y

WORKDIR /
# RUN git clone https://github.com/raysan5/raylib.git --depth 1
RUN wget https://github.com/raysan5/raylib/archive/refs/tags/4.5.0.zip
RUN ls
RUN unzip 4.5.0.zip
RUN mv raylib-4.5.0 raylib
WORKDIR /raylib
RUN ls

WORKDIR /raylib/src
RUN make PLATFORM=PLATFORM_WEB
RUN make install PLATFORM=PLATFORM_WEB

WORKDIR /
COPY raylib-build-web.sh .
RUN chmod +x raylib-build-web.sh

CMD [ "/raylib-build-web.sh" ]
```

**Comment by mihiic** [comment]
@raysan5 Not sure how viewing action helps me? Build of the library works fine for me too, problem is linking game against the library built from master branch. Sorry if I'm looking something wrong, but I only see building library files, not building sample against them.

On emscripten note:
I've tried with latest emscripten, in environment is the old one because I was trying to downgrade it

**Comment by raysan5** [comment]
@mihiic You are right, I confused the library building. It seems that symbol is not available on emscripten [`library_glfw.js`](https://github.com/emscripten-core/emscripten/blob/main/src/library_glfw.js).

**Comment by michaelfiber** [comment]
Here's the commit where they updated to the 3.3.8 version of the glfw header but did not implement that function:  https://github.com/emscripten-core/emscripten/commit/7cd514d84a1dc7ef966a875400d9c3f1e4ad6c95

**Comment by haedqu** [comment]
Why is this marked as closed? Is there any way to fix this without reverting to an old version, like maybe a simple way to add a dummy version of glfwGetError?

**Comment by raysan5** [comment]
@haedqu The issue was fixed with a possible solution, feel free to explore other options and send a PR if a better solution is found.
